configure

```diff
@@ -292,11 +292,16 @@ parser.add_option('--without-perfctr',
     dest='without_perfctr',
     help='build without performance counters')
 
+parser.add_option('--with-snapshot',
+    action='store_true',
+    dest='with_snapshot',
+    help=optparse.SUPPRESS_HELP)
```

Comment:
`optparse.SUPPRESS_HELP` means that `--with-snapshot` doesn't show when running `configure --help`, I think it would be helpful to add some help similar to the original help that was available for `--without-snapshot`.